Assets/Scripts/Models/World.cs

```diff
@@ -632,7 +632,7 @@ void LoadInventoryPrototypesFromFile(string inventoryXmlText)
         //furniturePrototypes["Door"].RegisterUpdateAction( FurnitureActions.Door_UpdateAction );
         //furniturePrototypes["Door"].IsEnterable = FurnitureActions.Door_IsEnterable;
 
-            //Logger.LogError("Did not find a 'Inventories' element in the prototype definition file.");
+        //Logger.LogError("Did not find a 'Inventories' element in the prototype definition file.");
```

Comment:
Do not know who asked you to 12 hours straight coding. Purpose of coding standard is that it is no my own.
But what the hell, you could leave as is.
But that you will the next who will read it and think who the hell wrote this...